lib/pbench/agent/tool_data_sink.py

```diff
@@ -1530,6 +1528,8 @@ def execute_action(self, client: str, data: Dict[str, Any]):
 
             self._from_tms_chan.close()
             self._to_logging_chan.unsubscribe()
+            self.logger.info("Tool Data Sink terminating")
```

Comment:
Bugger all, this log point results in a different function name in the test output which causes them to fail.  Just need a few tests updated.

Reply:
See https://github.com/Maxusmusti/pbench/pull/12.